src/packages.js

```diff
@@ -240,11 +246,12 @@ export default class Packages {
    * @return {Promise<object>}
    */
   _launchTheme(name, metadata) {
-    const preloads = (metadata.files || [])
-      .map(f => this.core.url(f, {}, ({
-        type: metadata.type,
-        ...metadata
-      })));
+    const preloads = metadataFilesToFilenames(
+      filterMetadataFilesByType(metadata.files, 'preload')
+    ).map(f => this.core.url(f, {}, ({
+      type: metadata.type,
```

Comment:
Hm. Looks like I did this.... but the `type: metadata.type` does't do anything here since it assigns `metadata` on the next line :joy: Not sure what I thought, but it would probably be one of those:

* `type: metadata.type || 'theme'`
* `type: 'theme'`

:joy:

Reply:
😂😂

Which one is correct? 😁

Reply:
It's should not be the first one because `metadata.type` is one of `['theme', 'icons', 'sounds']`.

Reply:
The second one is the "correct" one, but I'm not sure what I intended when I wrote this, hehe.